app/views/schedules/_new_assignments.html.erb

```diff
@@ -2,11 +2,11 @@
   <% assignment = @assignments.select {|assignment| (date == assignment.date) && (assignment.staff_id == staff.id)} %>
   <% unless assignment[0].nil? %>
     <div class="shift sqedule-border" style="background-color: <%= "#{"pink" if date.wday == 0}" %>">
-      <p><%= link_to "#{assignment[0].off == true ? 'OFF' : assignment[0].shift}", edit_staff_assignment_path(staff, assignment, date: date), id: "assignment_#{assignment[0].id}", class: "no-underline text-black p-6" %></p>
+      <p><%= link_to "#{assignment[0].off == true ? 'OFF' : assignment[0].shift}", edit_staff_assignment_path(staff, assignment, date: date), remote: true, id: "assignment_#{assignment[0].id}", class: "no-underline text-black p-6" %></p>
```

Comment:
Now that OFF is part of shift you can refactor this to just display assignment[0].shift:
`<%= link_to assignment[0].shift, edit_staff assignment_path... %>`